File: backend/Springboot/src/main/java/com/myApi/Springboot/Model/educacion.java

package com.myApi.Springboot.Model;

import javax.persistence.Basic;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Getter;
import lombok.Setter;

@Getter @Setter
@Entity
public class educacion {
        @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    long idEducacion;
    @Basic
    String institucion;
    String titulo;
    String carrera;
    String nota_promedio;
    int inicio ;
    int  fin;
    
  /*  @ManyToOne(mappedBy= "Persona")
    private Persona Persona*/

 
    

    public educacion(long idEducacion, String institucion, String titulo, String carrera, String nota_promedio, int inicio, int fin) {
        this.idEducacion = idEducacion;
        this.institucion = institucion;
        this.titulo = titulo;
        this.carrera = carrera;
        this.nota_promedio = nota_promedio;
        this.inicio = inicio;
        this.fin = fin;
    }

    
    public educacion() {
    }
  
}

File: src/app/entidades/educacion.ts


export class Educacion{
    id?:number;
    institucion:string;
    titulo:string;
    curso:string;
    inicio:any;
    fin:any;


constructor(institucion:string,titulo:string,curso:string,inicio:any,fin:any)
{
    
    this.institucion=institucion;
    this.titulo=titulo;
    this.curso=curso;
    this.inicio=inicio;
    this.fin=fin;
}
}